commands/download_mta_op_logs_command.go

```diff
@@ -74,30 +87,49 @@ func (c *DownloadMtaOperationLogsCommand) Execute(args []string) ExecutionStatus
 		return Failure
 	}
 
+	var operationIds []string
+
+	if hasMtaId(flags) {
+		operations, err := mtaClient.GetMtaOperations(&mtaId, formatLast(last), nil)
+		if err != nil {
+			ui.Failed(err.Error())
+			return Failure
+		}
+		for _, op := range operations {
+			operationIds = append(operationIds, op.ProcessID)
+		}
+	} else {
+		operationIds = append(operationIds, operationId)
+	}
+
+	for _, opId := range operationIds {
+		processDir := downloadDirName + string(os.PathSeparator) + defaultDownloadDirPrefix + opId + string(os.PathSeparator)
+		fmt.Println(processDir)
+		err = downloadLogsForProcess(opId, processDir, mtaClient, context)
```

Comment:
Here you have used processDir, down you have used downloadDirName.. Pick one name and use it...